tests/test_fairworkflow.py

```diff
@@ -55,9 +60,45 @@ def test_fn(x, y):
 
         assert workflow.validate() is True
 
-    def test_draw(self, tmp_path):
-        # Check for errors when calling draw()...
-        self.workflow.draw(filepath='test')
+    @mock.patch.dict('sys.modules', {'graphviz': None})
```

Comment:
Should we have a requirements_dev.txt that installs graphviz just to test that the visualization is created without error? I suppose it can't test if it looks correct, but it might pick up some exceptions

Reply:
Yeah, I think we should! Problem is that we need to install [graphvize (the non-python software](https://graphviz.org/), but I'll try to do that in CI.

Reply:
Ok, that was actually super easy. Turns out that we actually do not need the graphviz non-python software for the `display` method.